metaflow/runtime.py

```diff
@@ -216,6 +216,13 @@ def execute(self):
 
         except KeyboardInterrupt as ex:
             self._logger('Workflow interrupted.', system_msg=True, bad=True)
+            # Ignore new signals
+            import signal
+
+            def print_ignore(signum, frame):
+                self._logger('Shutdown in progress -- ignoring additional CTRL-C',
```

Comment:
It will be nice if the second `ctrl-C` instantly terminates the execution instead of blocking on the shutdown procedure.

Reply:
I think before that's what it did. I wanted to try to prevent multiple CTRL-C from preventing us from shutting down. I can remove this though if you feel that behavior is better.

Reply:
Yeah, let's remove it.